spec/factories.rb

```diff
@@ -1,4 +1,21 @@
 FactoryBot.define do
+  factory :payment_method do
+    name { Faker::Name.name }
+
+    left = Faker::Number.digit.to_i
+    right = Faker::Number.digit.to_i
+    rate { Faker::Number.decimal(left, right) }
```

Comment:
This factory is generating a decimal number with 6 elements before coma, and the rate must be at maximun 100.
So i think it's better to use this method rand(1.0..100.0).round(2).
It's going to generate a randon float number between 0 and 100 with maximum of 2 digits after coma.

Reply:
Rate can  be greater than 100. It can be diary, anualy...
If I use 'rand' into factory an alert message say something as 'It's not good practice.'